src/audio_worklet.js

```diff
@@ -144,6 +148,7 @@ class BootstrapMessages extends AudioWorkletProcessor {
     // Wasm Module and Memory objects.  After this we are ready to load in the
     // main application JS script, which the main thread will addModule()
     // to this scope.
+    globalThis.wasmWorkerArgs = arg['processorOptions'];
     globalThis.Module = arg['processorOptions'];
```

Comment:
These have the same value, is one of them not enough?

Reply:
This is the simplest way I could figure to make it work for now both in the audio worklet case and in the wasm worker case.

Audio worklet are just like wasm workers but they have this laternate creation path that currently lives in the `aw.js` file.   As a followup to this PR I hope to remove the aw.js file too, so this code here should no longer be needed once that lands.